src/hyp3lib/dem.py

```diff
@@ -0,0 +1,227 @@
+import json
+from collections.abc import Generator
+from pathlib import Path
+from subprocess import PIPE, run
+from tempfile import TemporaryDirectory
+
+from osgeo import gdal, ogr, osr
+
+from hyp3lib import DemError
+
+
+DEM_GEOJSON = '/vsicurl/https://asf-dem-west.s3.amazonaws.com/v2/cop30-2021-with-cop90-us-west-2-mirror.geojson'
+GEOID = '/vsicurl/https://asf-dem-west.s3.amazonaws.com/GEOID/us_nga_egm2008_1.tif'
+
+gdal.UseExceptions()
+ogr.UseExceptions()
+
+
+class GDALConfigManager:
```

Comment:
The context manager is useful outside DEM preparation; I'd suggest moving it to `hyp3lib.util`